### PR TITLE
Update the gameplay guide

### PR DESCRIPTION
**Pull request by necessarily-equal**

### CODE REVIEW

File: ui/help_gameplay.rml [hunk 1]
```diff
@@ -86,16 +86,17 @@
...
-				number of buildables will power down so that the other structures have
-				enough energy to remain operational.
-				<br />
-				<br />
 				Building a structure requires build points. Your team will start with a
 				small pool of build points but if you want to place a greater number of
 				buildables in an area, you can build a
 				<img class="emoticon" src="/emoticons/leech" />
 				mining structure to increase the available power and provide more build points.
+				Be wary that you will loose the build points if the mining structure are destroyed.
```
Comment (illwieckz) [inline ui/help_gameplay.rml:94]:
```diff
- structure are
+ structures are
```

File: ui/server_bots.rml [hunk 2]
```diff
@@ -45,14 +45,6 @@
...
 					<h3> Retreat </h3>
 				</row>
-				<row>
-					<input type="range" min="1" max="20" step="1" cvar="g_bot_numInGroup"/>
-					<h3> Group size</h3>
-					<p class="inline">
-						Current: <inlinecvar cvar="g_bot_numInGroup" type="number" format="%.0f"/>
-						<ilink onclick='Events.pushevent("exec reset g_bot_numInGroup", event)'> (reset) </ilink>
-					</p>
-				</row>
```
Comment (illwieckz) [inline ui/server_bots.rml:55]: I assume those changes are more for a PR like #53.

**Comment by necessarily-equal** [comment]
Ah indeed, I hadn't saw that one.

**Comment by illwieckz** [comment]
Though, see that comment: https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/pull/53#issuecomment-833428543
Because this is not modifying compiled code, and while I'm still writing the blog post, maybe the commit removing useless bot UI can sneak in…

**Comment by illwieckz** [comment]
Fun stuff:
- ffd88ca38f0cd476333ea5873f5ef09fc19f963e was because I noticed the mistake while doing a screenshot for the blog post.
- 062b8010c3bd67807b3bfda3a5500e11821e1401 was because it looked bad in a screenshot for the blog post.
- ffd88ca38f0cd476333ea5873f5ef09fc19f963e was because it contradicted what I wrote in the blog post.
- https://github.com/UnvanquishedAssets/res-players_src.dpkdir/commit/72392d0f9c2153108dd0ad0f80d11c183f7e51f2 was because I noticed the bug while doing a screenshot for the blog post, and that looked so bad…

🤦

**Comment by necessarily-equal** [comment]
I'd argue it makes sense to merge this, as it doesn't touch code (not 
even markup), so it really shouldn't break anything, and it can't be 
backported later on

**Comment by illwieckz** [comment]
This one is wanted.

**Pull request merged**